src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawer {
+    private String color = Color.WHITE.toString().toLowerCase();

```

Comment:
we shouldn't hardcode value in Figure class, so let's remove initialization of this field
```suggestion
    private String color;

```